admin-portal/src/pages/editPledge/components/pledgeSummary/pledgeSummary.tsx

```diff
@@ -34,7 +55,13 @@ export default function PledgeSummary({
       <div>
         <select onChange={handleStatusChange} value={pledge.status}>
           {Object.entries(Pledge.Status).map(([key, status]) => (
-            <option key={key} value={status}>
+            <option
+              key={key}
+              value={status}
+              disabled={pledgeStatusToAllowedStatuses[status].includes(
+                aidPackageStatus
+              )}
```

Comment:
It should `not` include, right?

```suggestion
              disabled={!pledgeStatusToAllowedStatuses[status].includes(
                aidPackageStatus
              )}
```